plugin/src/org/aavso/tools/vstar/external/plugin/ApacheCommonsLoessFitter.java

```diff
@@ -139,15 +140,16 @@ public String toString() {
 
 						double constCoeff = 0;
 
+						// We use formatGeneral because of a huge range
 						for (PolynomialFunction f : function.getPolynomials()) {
 							double[] coeffs = f.getCoefficients();
 							for (int i = coeffs.length - 1; i >= 1; i--) {
-								strRepr += "    " + coeffs[i];
-								strRepr += "t^" + i + "+\n";
+								strRepr += "    " + NumericPrecisionPrefs.formatPolyCoef(coeffs[i]);
+								strRepr += "*t^" + i + "+\n";
```

Comment:
Thanks @mpyat2. This was a hang-over from when functions were not VeLa, merely "descriptive" and I obviously did not test this transition properly for Loess models.